src/components/Navbar.js

```diff
@@ -2,23 +2,19 @@ import React, { Component } from 'react';
 import '../styles/Navbar.css';
 
 class Navbar extends Component {
-  constructor(props) {
-    super(props)
-  }
```

Comment:
This constructor wasn't needed – I got a warning in my console saying it wasn't necessary, so I removed.